### PR TITLE
EWS: Removed unused tasks and config page

### PR DESCRIPTION
**Pull request by kkrampa**
@gcapalbo @czue

### CODE REVIEW

**Comment by gcapalbo** [comment]
Is everything all set now with the email migration? I remember you mentioned some of Mike's scheduled reports didn't get migrated - was that issue resolved?

Also, just want to confirm that the purge of the connections user data has already been run?

**Comment by czue** [comment]
:+1: from me but will let you respond to @gcapalbo's questions

File: custom/ewsghana/views.py [hunk 1]
```diff
@@ -198,79 +170,26 @@ def post(self, request, *args, **kwargs):
...
-    delete_connections_field_task.delay(domain)
-    return HttpResponse('OK')
-
 
-@domain_admin_required
-@require_POST
-def delete_last_stock_data(request, domain):
-    delete_last_migrated_stock_data.delay(domain)
-    return HttpResponse('OK')
+class DashboardRedirectReportView(RedirectView):
```
Comment (dimagimon) [inline custom/ewsghana/views.py]: E303 too many blank lines (3)

**Comment by kkrampa** [comment]
@gcapalbo Field connections from user data has been already deleted. Regarding of Mike's scheduled reports I agreed with him that no action is necessary here.

**Comment by gcapalbo** [comment]
@kkrampa I'm still unclear about the emailed reports, will take to the email thread

**Pull request closed without merging**